### PR TITLE
Bump Go version

### PR DESCRIPTION
**Pull request by AlekSi**

### CODE REVIEW

**Review by EvgeniyPatlan — APPROVED** [review]

File: build/rpmbuild-docker/Dockerfile [hunk 1]
```diff
@@ -21,7 +21,7 @@ RUN yum install -y --setopt=skip_missing_names_on_install=False \
 RUN yum -y remove nodesource-release-el7-1.noarch
 RUN yum clean all && rm -rf /var/cache/yum
 
-ENV GO_VERSION 1.14.4
+ENV GO_VERSION 1.14.8
 ENV GO_CHECKSUM aed845e4185a0b2a3c3d5e1d0a35491702c55889192bb9c30e67a3de6849c067
```
Comment (AlekSi) [inline build/rpmbuild-docker/Dockerfile:25]: Oops. @Sudokamikaze please update checksum :)

Comment (Sudokamikaze) [reply build/rpmbuild-docker/Dockerfile:25]: Should I push into your branch or should I create another PR?

Comment (AlekSi) [reply build/rpmbuild-docker/Dockerfile:25]: Please update that PR and merge it once CI passes

Comment (AlekSi) [reply build/rpmbuild-docker/Dockerfile:25]: Actually, after manually confirming that it works – CI does not build rpmbuild

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1079-9e4b7fe
client docker - perconalab/pmm-client-fb:PR-1079-9e4b7fe
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1079-9e4b7fe.tar.gz

**Review by Sudokamikaze — APPROVED** [review]

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1079-41552c5
client docker - perconalab/pmm-client-fb:PR-1079-41552c5
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1079-41552c5.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1079-41552c5
client docker - perconalab/pmm-client-fb:PR-1079-41552c5
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1079-41552c5.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1079-41552c5
client docker - perconalab/pmm-client-fb:PR-1079-41552c5
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1079-41552c5.tar.gz

**Comment by Sudokamikaze** [comment]
Please confirm from your side so that we can merge it

**Review by EvgeniyPatlan — APPROVED** [review]

**Pull request merged**